capstone/target/step-96-2020-1/script.js

```diff
@@ -0,0 +1,470 @@
+// import $ from 'jquery'
+// import Popper from 'popper.js'
+// import Util from './util'
```

Comment:
This file seems mostly commented out. Why do we still need this?

Reply:
I deleted the whole file, but the target folder seems to contain duplicated that never updated.